packages/dds/tree/src/shared-tree-core/editManagerFormatCommons.ts

```diff
@@ -120,4 +120,26 @@ export const EncodedSharedBranch = <ChangeSchema extends TSchema>(tChange: Chang
 		noAdditionalProps,
 	);
 
+/**
+ * The format version for the EditManager.
+ */
+export const EditManagerFormatVersion = {
+	v1: 1,
+	v2: 2,
+	v3: 3,
+	v4: 4,
+	v5: 5,
+};
```

Comment:
I think the as const matters here since otherwise it all ends up types as `number`
```suggestion
} as const;
```